test/ObjectField_test.js

```diff
@@ -50,7 +50,10 @@ describe("ObjectField", () => {
     });
 
     it("should render a customized title", () => {
-      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+      const CustomTitleField = ({title, required}) => {
+        const legend = required ? title + '*' : title;
+        return <div id="custom">{legend}</div>;
+      };
```

Comment:
You're right. So my gut would be to revert to the old code for `CustomTitleField`, as the reader may really wonder why required label handling is implemented here, as I initially did :)